lang/main.json

```diff
@@ -212,6 +212,7 @@
     },
     "dialog": {
         "add": "Add",
+        "Allow": "Allow",
```

Comment:
General style nits:
- Maybe the translation key should be "allow" instead of "Allow"? The others start with a lowercase.
- Typo with "denys," should be "denies."
- Symbols in annotations are also being put in alphabetical order, so @public/@private before @returns. 
- The comment "Used to show hide the dialog on cancel." might be missing a word around "show hide." Maybe it's "show/hide." I realize it's also like this in AbstractDialog.
- You still could make actions right now for the calls to remote control within the dialog but you would leverage middleware for communicating outside of react/redux.
- I think Receiver.js's imports aren't in alphabetical order of file name. I think "/react/features" should be above "service/remotecontrol." That's the general pattern I've seen but maybe there's just inconsistency.
- Make sure to have a space after if.